public/index.html

```diff
@@ -5,6 +5,7 @@
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <link rel="shortcut icon" href="<%=htmlWebpackPlugin.options.FAVICON_URL%>" type="image/x-icon" />
+      <link rel="stylesheet" href=https://kit.fontawesome.com/d7e21cf75e.css crossorigin="anonymous">
```

Comment:
Why not adding this as part of the NPM requirements?

Reply:
This is for the pro icons that are paid. Scott only gave me the link to upload them, not the account to download from npm